doc/managed_identity_command_guideline.md

```diff
@@ -0,0 +1,49 @@
+## Overview
+This document provides a common design of the CLI command interface for supporting Managed Identity in Azure CLI. New commands should follow it while existing commands can stay the same before a deprecation plan.
+
+## Command interface
+
+### Enable managed identity during resource creation
+Use `--mi-system-assigned` to enable system assigned identity and `--mi-user-assigned` with space separated resource ids to add user assigned identities.
+
+```
+# <resource> can be acr, webapp, vm or any other resources that support managed identity
```

Comment:
`vm` already has managed identity support. Which should I use?

Reply:
Leave it as it is before there is a plan to change existing ones.